appveyor.yml

```diff
@@ -0,0 +1,26 @@
+version: 1.0.{build}
+max_jobs: 2
+image: Visual Studio 2015
+build_script:
+- cmd: >-
+    git submodule init
+
+    git submodule update
+
+    nuget restore ReactWindows\ReactNative.sln
+
+    set DEVENV="%VS140COMNTOOLS%\..\IDE\devenv"
+
+    %DEVENV% /build "Debug|x64" ReactWindows\ReactNative.sln
+
+    npm install -g npm
+
+    npm version
+
+    %DEVENV% /build "ReleaseBundle|x64" ReactWindows\ReactNative.sln
```

Comment:
>ReleaseBundle [](start = 21, length = 13)

I don't think we really need to build the *Bundle configs for react-native-windows, basically because the Bundle part only comes in to play at the application level (ReleaseBundle just maps to Release and DebugBundle just maps to Debug)